request_test.go

```diff
@@ -170,6 +184,19 @@ func TestVerbs(t *testing.T) {
 	}
 }
 
+func TestPostForm(t *testing.T) {
```

Comment:
I remember there was a testCase for all the httpVerbs at one place. Can you please check?

In that case we can remove this test, feel free to object me if you feel this testcase does more than checking the HTTPVerb only.

Reply:
Keeping test to check for correct header content type.

Reply:
I removed POSTFORM as a verb since it isn't an http method. It is just an extension of post.

Reply:
Required: Add one more assert where it will check if the HTTP Verb is of type POST.

Reply:
fixed